openapi/components/models/user.yml

```
@@ -0,0 +1,27 @@
+type: object
+title: User
+required:
+  - id
+  - username
+  - displayName
+  - bio
+  - isPrivate
+  - createdAt
+  - updatedAt
+properties:
+  id:
+    type: string
```

```suggestion
  id:
    type: string
    format: uuid
```

Let's add the `format` keyword to indicate that the `id` field should follow the UUID format.